timeseries/src/autogluon/timeseries/predictor.py

```diff
@@ -775,23 +769,23 @@ def predict(
         Parameters
         ----------
         data : Union[TimeSeriesDataFrame, pd.DataFrame, Path, str]
-            Time series data to forecast with.
-
-            If ``known_covariates_names`` were specified when creating the predictor, ``data`` must include the columns
-            listed in ``known_covariates_names`` with the covariates values aligned with the target time series.
+            Historic time series data for which the forecast needs to be made.
```

Comment:
Nit: https://www.merriam-webster.com/grammar/everything-youve-ever-wanted-to-know-about-historic-and-historical

Reply:
Thanks, this is probably the 5th time that I realize that I'm using the wrong term, but somehow this knowledge gets magically erased from my mind 😅